Microsoft.Dynamics365.UIAutomation.Api.UCI/WebClient.cs

```diff
@@ -3344,6 +3345,54 @@ internal BrowserCommandResult<bool> SelectLookupNewButton()
             });
         }
 
+        internal BrowserCommandResult<IReadOnlyCollection<FormNotification>> GetFormNotifications()
+        {
+            return Execute(GetOptions($"Get all form notifications"), driver =>
+            {
+                List<FormNotification> notifications = new List<FormNotification>();
+
+                if (!driver.TryFindElement(By.XPath(AppElements.Xpath[AppReference.Entity.FormNotifcationBar]),
+                    out var notificationBar))
+                {
+                    return notifications;
+                }

```

Comment:
Replace TryFindElement with WaitUntilVisible (2 secs) just for the case that the notification requiere some time to fade-out.
```csharp
    var notificationBar = driver.WaitUntilVisible(By.XPath(AppElements.Xpath[AppReference.Entity.FormNotifcationBar]), 2.Secounds());
    if(notificationBar == null)
        return notifications;
```

Reply:
Never had any issues with that, however it makes sense. Better safe than sorry.